core/lib/spree/testing_support/common_rake.rb

```diff
@@ -32,6 +32,6 @@
   task :seed do |_t, _args|
     puts "Seeding ..."
 
-    sh "bundle exec rake db:seed RAILS_ENV=test"
+    sh "rake db:seed RAILS_ENV=test"
```

Comment:
Why, is `bundle exec` causing any problems?

Removing this could be cause of conflicting gem errors, using the `bin/rake` (or even better `bundle exec`, because of a possible `spring` binstub) would solve this, no?